hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -129,9 +129,12 @@ public void incrementCount(List<Long> txIDs) throws IOException {
         DeletedBlocksTransaction block =
             scmMetadataStore.getDeletedBlocksTXTable().get(txID);
         if (block == null) {
-          // Should we make this an error ? How can we not find the deleted
-          // TXID?
-          LOG.warn("Deleted TXID {} not found.", txID);
```

Comment:
Let's keep this rare error case as WARN. Other changes LGTM.

Reply:
@xiaoyuyao I can see this warning message being printed a lot of times during deletion. It can easily fill up SCM log during deletion. How about we change the log level and investigate the reason in a separate jira?
Since SCM is processing thousand of transactions and it resends these transactions to a particular datanode. Then this log can be printed thousands of times.

Reply:
Agree with the analysis. Let's investigate the cause in a separate JIRA. +1 for merging the PR as-is.